src/gvz_api/utils.py

```diff
@@ -141,21 +159,6 @@ def best_match(self, region_name, region_type=None):
                 results_literal.append(region)
         if len(results_literal) == 1:
```

Comment:
Very cool GVZ API :rocket:
Coincidentally I had just a look at the code, too.
I found that saving a region called "Augsburg" with administrative division "Rural district" ("Landkreis")
displays the logger message: "GVZ API did not find type match for 'Augsburg'",
because the `results_literal` list contained two entries: "Kreisfreie Stadt (Kreisebene)" and "Landkreis".
Maybe we would need here some additional filters?

Reply:
If I understood the [constants](https://github.com/Integreat/gemeindeverzeichnis-django/blob/main/src/gvz/constants.py) defined in the GVZ API correctly, the mapping should be the following:
```
Name             | Django         | GVZ

Kreis            | DISTRICT       | CT_ABSTRACT (40) or CT_COUNTY (43)
Landkreis        | RURAL_DISTRICT | CT_RURAL_COUNTY (44)
Kreisfreie Stadt | URBAN_DISTRICT | CT_URBAN_MUNICIPALITY (41) or MP_URBAN_MUNICIPALITY (61)
```
But in the [utils](https://github.com/Integreat/integreat-cms/blob/7b86675a13e75dfac0798ddcd7411e17b9fb956b/src/gvz_api/utils.py#L106-L127), this is defined a bit different :thinking: 

I think the optimal way would be if we'd find a definition we can all agree on and use the same one for Django and for the GVZ API? Then we wouldn't need a hardcoded mapping between the two systems... which source did you use to create the constants for the GVZ API?

Reply:
The names come from https://www.destatis.de/DE/Themen/Laender-Regionen/Regionales/Gemeindeverzeichnis/Administrativ/beschreibung-gebietseinheiten.pdf?__blob=publicationFile. However, the translations into English constants are a bit tricky/not perfect.

With carefully selecting all required objects from the GVZ, we could get around the mapping. However, the official values are very redundant right now, and I'm not sure if we really like to do this. Right now I'd recommend to keep the mapping.

Reply:
Ok, I see. Also the issue with e.g. "Landkreis Augsburg" persists (that no unambiguous match is found even if there are not multiple Landkreise with the name "Augsburg"), but if there is no easy fix to it, we should probably open a new issue for more refined constants and a better mapping...